queries/gitattributes/highlights.scm

```diff
@@ -42,12 +42,12 @@
 (macro_def
   macro_name: (_) @property)
 
-[
-  (pattern_negation)
-  (redundant_escape)
-  (trailing_slash)
-] @error
+; [
+;   (pattern_negation)
+;   (redundant_escape)
+;   (trailing_slash)
+; ] @error
```

Comment:
These are explicitly parsed as errors, similar to some vim syntax files. Do we want to keep them or should I remove them from the grammar?

Reply:
Maybe we could change them to `@text.warning`?

Reply:
I'll go with whatever you and clason choose